mkosi/__init__.py

```diff
@@ -6285,6 +6298,49 @@ def run_ssh(args: CommandLineArguments) -> None:
         )
 
 
+def generate_secure_boot_key(args: CommandLineArguments) -> NoReturn:
+    """Generate secure boot keys using openssl"""
+    args.secure_boot_key = args.secure_boot_key or "./mkosi.secure-boot.key"
+    args.secure_boot_certificate = args.secure_boot_certificate or "./mkosi.secure-boot.crt"
+
+    keylength = 2048
+    expiration_date = datetime.date.today() + datetime.timedelta(int(args.secure_boot_valid_days))
+    cn = expand_specifier(args.secure_boot_common_name)
+
+    for f in (args.secure_boot_key, args.secure_boot_certificate):
+        if os.path.exists(f) and not args.force:
+            die(
+                f"{f} already exists.\n"
+                "If you are sure you want to generate new secure boot keys\n"
+                f"remove {args.secure_boot_key} and {args.secure_boot_certificate} first."
```

Comment:
I'd make this a multiline string and use dedent

Reply:
Will do, but as long as it's only a few lines, I tend to prefer implicit concatenation instead of the additional level of indentation.